straxen/plugins/nveto_recorder.py

```diff
@@ -20,9 +20,7 @@
                  help="Number of lone hits to be stored per channel for diagnostic reasons."),
     strax.Option('channel_map', track=False, type=immutabledict,
                  help="frozendict mapping subdetector to (min, max) "
-                      "channel number."),
-    strax.Option('n_nveto_pmts', type=int, track=False,
-                 help='Number of nVETO PMTs')
+                      "channel number.")
```

Comment:
extremely minor, if you have a list like this. Ending with `,` will keep the git history cleaner as now it will change this line whereas you did not change anything actually.

Reply:
Will change it